app/models/miq_provision_virt_workflow.rb

```diff
@@ -1036,10 +1036,23 @@ def ws_vm_fields(values, fields)
     ws_network_fields(values, fields, data)
     ws_customize_fields(values, fields, data)
     ws_schedule_fields(values, fields, data)
+    ws_environment_fields(values, fields, data)
 
     data.each { |k, v| _log.warn "Unprocessed key <#{k}> with value <#{v.inspect}>" }
   end
 
+  def ws_environment_fields(values, _fields, data)
```

Comment:
Is the line added above the only consumer of this method? Could the `_fields` arg be removed if that is the case?